incubator/goldfish/values.yaml

```diff
@@ -3,16 +3,14 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: quay.io/tuannvm/goldfish
-  tag: v0.7.3
-  pullPolicy: IfNotPresent
+  repository: caiyeon/goldfish
+  tag: 0.9.0
+  pullPolicy: Always
```

Comment:
Keep `IfNotPresent`.

Reply:
Yep, I'm planning to help out with that stuff.  Updated to use the `IfNotPresent` policy.